app/services/end_of_cycle_timetable.rb

```diff
@@ -72,14 +92,7 @@ def self.next_cycle_year
 
   def self.schedules
     {
-      real: {
-        apply_1_deadline: Date.new(2020, 8, 24),
-        stop_applications_to_unavailable_course_options: Date.new(2020, 9, 7),
-        apply_2_deadline: Date.new(2020, 9, 18),
-        find_closes: Date.new(2020, 10, 3),
-        find_reopens: Date.new(2020, 10, 6),
-        apply_reopens: Date.new(2020, 10, 13),
-      },
+      real: CYCLE_DATES[2021],
```

Comment:
`2021` here effectively controls `RecruitmentCycle.current_year`, give or take some adjustment for the transition before `find_reopens`. Would it be better to create and refer to a `RecruitmentCycle::CURRENT_YEAR` here, so that adjusting the year happens in one place, `RecruitmentCycle`?

Reply:
It feels like this whole thing needs some love. It looks like EndOfCycleTimetable is actually the source of truth, and RecruitmentCycle reads from it to determine which year we are in. I think that makes sense, especially if we rename EndOfCycleTimetable to CycleTimetable (since it's not especially related to the end of a cycle)

Reply:
That's fine but even if `CycleTimetable` remains the source of truth for which year we're in, having `2021` hardcoded inside the values of a hash called `.schedules` seems a bit hidden away. I'd rather have this defined at the top of the file and named according to its significance! Not a huge deal, I just don't like where this sits at the moment.

Reply:
Ah that's a good point - will extract to a named constant